os/board/rtl8721csm/src/component/os/tizenrt/tizenrt_service.c

```diff
@@ -1185,6 +1185,30 @@ int rtw_printf_info(const char *format,...)
 	return ret;
 }
 
+/* Blocks the currently running task, need to called within the currently running task.
```

Comment:
Please change "need to called within"  to "need to be called within", I think it sounds better.